migrations/env.py

```diff
@@ -0,0 +1,90 @@
+from __future__ import with_statement
```

Comment:
We definitely don't need this line; `with` statements became mandatory in Python 2.6:

```
>>> from __future__ import with_statement
>>> with_statement.getMandatoryRelease()
(2, 6, 0, 'alpha', 0)
```